bundles/framework/timeseries/view/TimeSeriesRange/TimeSeriesRangeControl.jsx

```diff
@@ -0,0 +1,66 @@
+import { Controller } from 'oskari-ui/util';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Background, Header, Col, ColFixed, Row, YearInput, YearRangeSlider } from './styled';
+
+export const TimeSeriesRangeControl = ({ controller, title, start, end, value, dataYears, isMobile }) => {
+    const [startValue, endValue] = value;
+    const marks = {
+        [start]: start,
+        [end]: end,
+    };
+    for (let i = start + 1; i < end; i++) {
+        if (i % 10 === 0) {
+            marks[i] = i;
+        }
+    }
```

Comment:
This (marks) could be an impl detail that is implemented in YearRangeSlider, but I'm really nitpicking here. This looks really nice for a first version.